batch-processing/pom.xml

```diff
@@ -200,12 +203,32 @@
                         <groupId>org.wildfly.plugins</groupId>
                         <artifactId>wildfly-maven-plugin</artifactId>
                         <configuration>
-                            <discover-provisioning-info>
```

Comment:
why are we going back to directly using galleon layers, instead of glow's discovery? This seems a mistake

Reply:
Maybe you are just doing copy paste of outdated (in the sense of not using glow) configuration? You can use glow and pack a script as shown at https://github.com/wildfly/quickstart/blob/82ceea0a283cd1b7a2d1bdfd51a8c41b23e8b182/opentelemetry-tracing/pom.xml#L161